service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/StandardDirectionsOrderControllerDateOfIssueMidEventTest.java

```diff
@@ -83,9 +82,7 @@ void shouldRemoveEmptyPreparedStandardDirectionOrderDocumentIfPresent() {
         }
 
         @Test
-        void shouldSetJudgeAndLegalAdvisorWhenSendNoticeOfProceedingsViaSDOIsToggledOn() {
```

Comment:
same
    @MockBean
    FeatureToggleService featureToggleService;
to be removed